utils/docker/build_docker_image.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#===- llvm/tools/pstore/utils/docker/build_docker_image.sh ----------------===//
```

Comment:
This path (and the use of the LLVM license below rather than the pstore license) point to this commit belonging to the llvm-prepo project rather than pstore. The docker image contains the compiler with some supporting pstore tools.